### DISCUSSION TITLE
export single pages from cbz

### DISCUSSION THREAD

**Discussion by eadmaster** [sumatrapdfreader/sumatrapdf#2765] [Ideas]
i'd like to export/save the current page while viewing a comic book in cbz format.

I know i can open the cbz with an archive viewer, but there is no easy way to locate the file corresponding to the current page in SumatraPDF.

**Comment by GitHubRulesOK** [comment]
Hmm 
there is an open issue about filenames in folders but cbx is numbers by page position so not as easy to say the two match for extract, due to sorting, which notoriously is different in windows apps such as tar (the new native unzipper)
![image](https://user-images.githubusercontent.com/12754913/176332157-337dec25-c3a6-4f0b-b584-8001d7f54b0f.png)

By far the easiest is to right click and save as image it will be pixel size perfect whatever scale
![image](https://user-images.githubusercontent.com/12754913/176332545-3f26078e-8e01-4814-892d-feed44c276f9.png)
